controller/build.gradle

```diff
@@ -6,11 +6,14 @@ dependencies {
     api(project(":entities"))
     api(project(":spi"))
     api(project(":core"))
+    api(project(':rateLimiter'))
 
+    implementation(project(":common"))
     implementation("io.vertx:vertx-micrometer-metrics")
     implementation('com.fasterxml.jackson.core:jackson-databind')
     implementation("org.apache.commons:commons-lang3")
     testImplementation(testFixtures(project(":entities")))
     testImplementation(testFixtures(project(":common")))
     testImplementation("org.awaitility:awaitility:4.2.1")
+    testImplementation(project(':server'))
```

Comment:
controller should not be depending on server module. (even for tests)

Reply:
moved tests to server module as it depends on AppConfigs